### ISSUE TITLE
Behavior when data is ingested into ES index - with timestamp that is B4 the time of the last request

### ISSUE THREAD

**Issue by mrkstate** [sivasamyk/logtrail#278]
There are circumstances where data is ingested into an ES indice - that have a timestamp that is before the lasttimestamp within the browser.  That data will never appear in the scrolling data - as it is not brough forward in any subsequent requests.

For example:

The browser is started and the initial request is made for data @ 3:00pm.  All the data (within the parameters in the logtrail,json) is brought into the display.

Another request is made N number of seconds later.  During that time - 30 new records were ingested into the ES indice.  10 of them timestamp just b4 3:00 pm (as it took a bit for the data to be forwarded thru the infrastructure) and 20 with a timestamp after 3:00 pm.

The new data (20 records) is shown - the other 10 are not.

Our setting for es_index_time_offset_in_seconds is set to 0 (ZERO).

We tested changing the value to "60" for 1 minute.  We started to see older records come back - however - the order within the UI was appended at the end and not injected - in place/proper sequence.

Is this the expected behavior?

Thanks !!

**Comment by sivasamyk** [comment]
Yes. Logtrail does not sort the events. The sorting is done at elasticsearch server. Logtrail ensures there are no duplicate events displayed. Are you using logstash to ingest data into ES? If so you can map timestamp field to logstash receive timestamp, and add application timestamp as an additional field in `message_format`

**Comment by ShaunK** [comment]
@sivasamyk I experienced this same issue, and was able to add sorting when new updates came in so that everything would be in order. Is that something you'd be willing to accept as a PR if I was to clean it up and submit it?
